tests/integration/seccomp-notify.bats

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_seccompagent
+	setup_seccompagent
+
+	setup_busybox
+}
+
+function teardown() {
+	teardown_seccompagent
+	teardown_bundle
+}
+
+# Create config.json template with SCMP_ACT_NOTIFY actions
+# $1: command to run
+# $2: noNewPrivileges (false/true)
+# $3: list of syscalls
+function scmp_act_notify_template() {
+	# The agent intercepts mkdir syscalls and creates the folder appending
+	# "-bar" (listenerMetadata below) to the name.
+	update_config '.process.args = ["/bin/sh", "-c", "'"$1"'"] |
+	.process.noNewPrivileges = '"$2"' |
+	.linux.seccomp = {
+		"defaultAction":"SCMP_ACT_ALLOW",
+		"listenerPath": "'"$SECCCOMP_AGENT_SOCKET"'",
+		"listenerMetadata": "bar",
+		"architectures":["SCMP_ARCH_X86","SCMP_ARCH_X32"],
```

Comment:
Is `SCMP_ARCH_X86_64` intentionally excluded here?

Reply:
No, added it. Thanks!